src/main/resources/META-INF/reaktivity/sse.idl

```diff
@@ -37,5 +37,10 @@ scope sse
             string id;
             string type;
         }
+        
+        struct SseEndEx
+        {
+            string lastEventId;
```

Comment:
This should be named `id` not `lastEventId` as it is just the `SseEndEx` variation of `SseDataEx`.

Reply:
Fixed